spec/index.html

```diff
@@ -297,15 +297,48 @@ <h3>IRIs</h3>
     <h3>Literals</h3>
 
     <p><a data-cite="RDF12-CONCEPTS#dfn-literal">Literals</a>
-    are basic values that are not IRIs. Examples of literals include
-    strings such as "La Joconde", dates such as "the 4th of July, 1990"
-    and numbers such as "3.14159".
-    Literals are associated with a <a data-cite="RDF12-CONCEPTS#dfn-datatype">datatype</a>
-    enabling such values to be parsed and interpreted correctly.
-    String literals can optionally be associated with a <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tag</a>.
-    For example, "Léonard de Vinci" could
-    be associated with the "fr" language tag and "李奥纳多·达·文西"
-    with the "zh" language tag.</p>
+    are used to encode values, rather than identifying them by IRIs. Examples
+    of literals include strings such as "La Joconde", dates such as
+    "1990-07-04" (the 4th of July, 1990) and numbers such as "3.14159".
```

Comment:
```suggestion
    of literals include strings such as "La Joconde", dates such as
    "1990-07-04" (which could also be expressed as a string like
    "the fourth of July, 1990"), and numbers such as "3.14159".
```

Reply:
I'm not sure of this; if made more explicit I'd rather explain this as the ISO-8601/W3C date-encoded form of that date... I don't mind more wordsmithing here; but can also merge without further changes if you consider this a minor concern @TallTed .

Reply:
If I understand you correctly, you'd prefer this (with which I'm fine) --
```suggestion
    of literals include strings such as `"La Joconde"`, ISO-8601/W3C date-encoded
    dates such as `"1990-07-04"`, and numbers such as `"3.14159"`.
```